server/utils/middleware.js

```diff
@@ -28,3 +30,24 @@ export function ifNoUser401(req, res, next) {
   }
   return res.status(401).end();
 }
+
+export function flashIfNotVerified(req, res, next) {
+    const user = req.user;
+    if (!user) {
+      return next();
+    }
+    const email = req.user.email;
+    const emailVerified = req.user.emailVerified;
+    if (!email) {
+      req.flash('info', { msg:
+         dedent `Please update your email address when you get a moment in
+                  your <a href="\settings"> Settings Page.</a>`
+      });
+    } else if (!emailVerified) {
+      req.flash('info', { msg:
+         dedent `We have your email address with us, but its not yet verified.
+                  Please follow the link we sent you, when you get a moment.`
```

Comment:
"Please check your email. We sent you a link that you can click to verify your email address." revised copy.